README.md

```diff
@@ -7,48 +7,36 @@
 
 [![Build Status](https://api.travis-ci.org/appium/appium-xcuitest-driver.png?branch=master)](https://travis-ci.org/appium/appium-xcuitest-driver)
 
-*Note*: Issue tracking for this repo has been disabled. Please use the [main Appium issue tracker](https://github.com/appium/appium/issues) instead.
+Appium XCUITest Driver is a combined solution, which allows to perform automated black-box testing of iOS and tvOS native applications and Safari web views.
+The native testing is based on Apple's [XCTest](https://developer.apple.com/documentation/xctest) framework and the fork of Facebook's [WebDriverAgent](https://github.com/appium/WebDriverAgent) server (the [original](https://github.com/facebookarchive/WebDriverAgent) project is not supported anymore).
+Web views communication is done via [Webkit remote debugger protocol](https://github.com/appium/appium-remote-debugger). Real devices communication is ensured by [appium-ios-device library](https://github.com/appium/appium-ios-device).
+Simulators communication is ensured by [appium-ios-simulator](https://github.com/appium/appium-ios-simulator) library.
 
+In the native mode the driver operates in scope of [WebDriver W3C protocol](https://w3c.github.io/webdriver) with several platform-specific extensions. Web views communication only supports the obsolete [JWP protocol](https://webdriver.io/docs/api/jsonwp.html).
 
-## Known issues
+*Note*: Issue tracking for this repo has been disabled. Please use the [main Appium issue tracker](https://github.com/appium/appium/issues) instead.
```

Comment:
just FYI that when we move to Appium 2.0 we'll be able to turn issue tracking back on here so that each of the drivers can manage their own issues.